src/main/java/com/carrotcreative/cream/loaders/single/SingleLoader.java

```diff
@@ -25,7 +25,7 @@ public void loadSelf(final T identifier, final SingleLoaderCallback callback){
         handleInitialLoad(identifier, callback);
     }
 
-    protected void handleInitialLoad(final T identifier, final SingleLoaderCallback callback)
+    void handleInitialLoad(final T identifier, final SingleLoaderCallback callback)
```

Comment:
The default protection level in java is `package-private` -- which is a bad idea in this case.  A child singleLoader outside of this package can't override this method if that's the case!